docs/reference/api/docker_remote_api.md

```diff
@@ -130,6 +130,7 @@ This section lists each version from latest to oldest.  Each listing includes a
   containers that are tasks (part of a service in swarm mode).
 * `POST /containers/create` now takes `StopTimeout` field.
 * `POST /services/create` and `POST /services/(id or name)/update` now accept `Monitor` and `MaxFailureRatio` parameters, which control the response to failures during service updates.
+* `GET /networks/(name)` now returns `Created`.
```

Comment:
Thanks @AkihiroSuda . I do like the PR, while I think maybe here is some ambiguous. First I see here, I thought return a status code of Created. I have not thought about adding a field `Created` in response.